DeepCat.js

```diff
@@ -642,9 +642,17 @@
 		}
 	}
 
-	$( function() {
-		deepCatMain();
-	} );
+	if( mw.loader.getState( 'ext.gadget.DeepCat' )  !==  'ready' ) {
+		mw.loader.state( 'ext.gadget.DeepCat', 'ready' );
+		mw.loader.using(
+			[
+				'mediawiki.api.messages',
+				'mediawiki.cookie',
+				'mediawiki.util',
+				'mediawiki.jqueryMsg'
+			],
+			function() { $( deepCatMain ); } );
```

Comment:
it would be more readable if closing parenthesis and a colon were in their own line. I'll change this. Apart from that ok!